econsa/tests/test_sampling.py

```diff
@@ -53,40 +61,27 @@ def test_cond_mvn():
     np.testing.assert_allclose(cond_cov, r_cond_cov)
 
 
-def test_cond_mvn_exception_given():
-    """Test cond_mvn raises exceptions when invalid `given_ind` or `given_value` is passed."""
-    mean, cov, dependent_ind, given_ind, given_value = get_strategies(
-        "cond_mvn_exception_given",
-    )
+def test_cond_mvn_given_valid():
+    """ Valid case: only `given_ind` is empty or both `given_ind` and `given_value` are empty. """
+    cond_mvn(*get_strategies("cond_mvn_given_valid"))
 
-    n = cov.shape[0]
-    if n % 3 != 0:
-        # Valid case: only `given_ind` is empty or both `given_ind` and `given_value` are empty
-        cond_mvn(mean, cov, dependent_ind, given_ind, given_value)
-    else:
-        # `given_value` is empty or does not align with `given_ind`
-        with pytest.raises(ValueError) as e:
-            cond_mvn(mean, cov, dependent_ind, given_ind, given_value)
-        assert "lengths of given_value and given_ind must be the same" in str(e.value)
-
-
-def test_cond_mvn_exception_cov():
-    """Test cond_mvn raises exceptions when invalid `cov` is passed."""
-    mean, cov, dependent_ind, given_ind, given_value = get_strategies(
-        "test_cond_mvn_exception_cov",
-    )
-
-    n = cov.shape[0]
-
-    if n % 3 != 0 and n % 2 != 0:
-        # `cov` is negative definite matrix
-        with pytest.raises(ValueError) as e:
-            cond_mvn(mean, cov, dependent_ind, given_ind, given_value)
-        assert "cov is not positive-definite" in str(e.value)
-    elif n % 2 != 0:
-        # `cov` is not symmetric
-        with pytest.raises(ValueError) as e:
-            cond_mvn(mean, cov, dependent_ind, given_ind, given_value)
-        assert "cov is not a symmetric matrix" in str(e.value)
-    else:
-        cond_mvn(mean, cov, dependent_ind, given_ind, given_value)
+
+def test_cond_mvn_given_invalid():
+    """`given_value` is empty or does not align with `given_ind`."""
+    with pytest.raises(ValueError) as e:
+        cond_mvn(*get_strategies("cond_mvn_given_invalid"))
+    assert "lengths of given_value and given_ind must be the same" in str(e.value)
+
+
+def test_cond_mvn_cov_negative():
+    # `cov` is negative definite matrix
+    with pytest.raises(ValueError) as e:
+        cond_mvn(*get_strategies("cond_mvn_cov_negative"))
+    assert "cov is not positive-definite" in str(e.value)
+
+
+def test_cond_mvn_cov_asymmetric():
```

Comment:
Please just add one full sentence at least as docstring, with capitalized first workd and then a `.` in the end.